.github/workflows/selenium_tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Selenium Tests With Screenshots
+
+on:
+  pull_request:
+    types: [ labeled, synchronize, opened, reopened ]
+
+permissions:
+   contents: read
+
+jobs:
+  Run_Selenium_tests_with_screenshots:
+    if: contains(github.event.pull_request.labels.*.name, 'enhancement')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+          cache: 'pip'
+          cache-dependency-path: 'tests/requirements/selenium.txt'
+      - name: Install and upgrade packaging tools
+        run: python -m pip install --upgrade pip setuptools wheel
+      - run: python -m pip install -r tests/requirements/selenium.txt .
+
+      - name: Run selenium tests for current PR
+        working-directory: ./tests/
+        run: python runtests.py --selenium=chrome --headless --parallel 2 admin_changelist admin_inlines admin_widgets forms_tests
```

Comment:
This is stating which tests to run because without it I was getting database threading errors
```
django.db.utils.DatabaseError: DatabaseWrapper objects created in a thread can only be used in that same thread. The object with alias 'default' was created in thread id 139881739424192 and this is thread id 139880491378240.
```

I had no idea where this was coming from but this [blog article](https://www.dimagi.com/blog/debugging-a-thread-related-test-failure/) implied that something is being imported in the test discovery and causing this issue. By only stating the test folders which have selenium tests, this went away but it is not ideal as it's a people are unlikely to know they must update this when adding selenium tests to new test folders.
I can try to find a better solution but if anyone knows how to fix this that would be lovely.